service-base/src/main/java/org/eclipse/hono/service/management/tenant/Tenant.java

```diff
@@ -41,6 +41,10 @@
     @JsonInclude(Include.NON_EMPTY)
     private List<Adapter> adapters = new LinkedList<>();
 
+    @JsonProperty("minimum-message-size")
```

Comment:
IMHO this should be a constant declared in `RegistryManagementConstants` ...

Reply:
Done. Pls take a look.